apps/src/block_utils.js

```diff
@@ -794,26 +781,6 @@ const STANDARD_INPUT_TYPES = {
   }
 };
 
-/**
- * Given a type string for a field input, returns an appropriate change handler function
- * for that type, which customizes the input field and provides validation on blur.
- * @param {Blockly} blockly
- * @param {string} type
- * @returns {?function}
- */
-function getFieldInputChangeHandler(blockly, type) {
```

Comment:
I really like that the changes to this file are mostly red/negative and that we're moving some of this outdated logic to the cdo wrapper instead.